packages/opentelemetry-cloud-monitoring-exporter/src/monitoring.ts

```diff
@@ -111,7 +116,13 @@ export class MetricExporter implements IMetricExporter {
         );
       }
     }
-    this._sendTimeSeries(timeSeries);
+
+    for (const batchedTimeSeries of partitionList(
+      timeSeries,
+      MAX_BATCH_EXPORT_SIZE
+    )) {
+      await this._sendTimeSeries(batchedTimeSeries);
```

Comment:
Added an `await` here for the async code in `_sendTimeSeries` to be properly called and spied on in tests.

I was wondering if there was a preferred testing practice so we don't have to wait here, or if it's ok to wait so we can handle if an error is thrown?